Frontend/src/components/rankings/ResultsTable.vue

```diff
@@ -2,16 +2,19 @@
   <div class="card" v-if="ranking">
     <div class="card-body">
       <h6 class="card-title">
-        <i class="bi bi-trophy" :class="ranking.rankingClass"></i>
-        {{ ranking.rankingName }}
+        <i class="bi bi-trophy" :class="ranking.name"></i>
```

Comment:
Ich denke hier würden die Daten aus seasonDetail.ratingClasses.<<CLASS>>.description mehr Sinn ergeben. 
Vielleicht müssen wir die noch leicht anpassen. Schau doch mal in seasonDetails.json rein.

Dann würde ich die zurückgeben, anstatt den Namen des Object-Props.

Reply:
Genau, das wollte ich, hatte das nicht gefunden.

Reply:
Willst du dir die dann aus den seasonDetails holen?

Reply:
Es wäre geiler, wenn es schon im ratingClass Array wäre. Wenn das easy geht. Ansonsten wird das ne aufwändige hin und her reichere von Daten im Frontend.

Reply:
Vielleicht das Object erweitern um readableName